### DISCUSSION TITLE
Limiting the size of URL query parameters

### DISCUSSION THREAD

**Discussion by ririsoft** [tokio-rs/axum#1723] [Q&A]
Hello,

Does Axum Query extractor limits the size of URL query parameters ?
If yes is there any way to customise the size limit ?
If no, what would the "Axum Idiomatic way" be to restrict the size of URL query parameter and return an HTTP error ?

Thanks in advance for your help.
Cheers,
Riri.

**Comment by davidpdrsn** [comment]
I do believe hyper enforces a limit but not sure what it is or how to customize it. Try asking in the hyper discord server or GitHub repo.

**Reply by ririsoft** [reply]
Thank you @davidpdrsn, I posted the question as suggested, let's see what comes out : https://discord.com/channels/500028886025895936/670880858630258689/1068078461513642044
